src/main/java/online/kingdomkeys/kingdomkeys/client/gui/synthesis/SynthesisCreateScreen.java

```diff
@@ -264,10 +264,10 @@ protected void renderSelectedData(GuiGraphics gui, int mouseX, int mouseY, float
 			
 			if(RecipeRegistry.getInstance().containsKey(selectedRL)) {
 				Recipe recipe = RecipeRegistry.getInstance().getValue(selectedRL);
-				gui.drawString(minecraft.font, Utils.translateToLocal(Strings.Gui_Shop_Buy_Cost)+":", 2, -20, Color.yellow.getRGB());
-				String line = recipe.getCost()+" "+Utils.translateToLocal(Strings.Gui_Menu_Main_Munny);
+				gui.drawString(minecraft.font, Utils.translateToLocal(Gui_Shop_Buy_Cost)+":", 2, -20, Color.yellow.getRGB());
+				String line = recipe.getCost()+" "+Utils.translateToLocal(Gui_Menu_Main_Munny);
```

Comment:
Strings.Gui_Menu_Main_Munny
Probably could include the cost in the translation too